.eslintrc

```diff
@@ -80,106 +80,106 @@
-        "Autocomplete": true,
-        "BLUE_BUTTON": true,
-        "bootstrap": true,
-        "browser": true,
-        "browserAction": true,
-        "CHECK_UPDATE_NEVER": true,
-        "CHECK_UPDATE_ONE_MONTH": true,
-        "CHECK_UPDATE_ONE_WEEK": true,
-        "CHECK_UPDATE_THREE_DAYS": true,
-        "cloneInto": true,
-        "createResult": true,
-        "createStylesheet": true,
-        "DatabaseState": true,
-        "debugLogMessage": true,
-        "EXTENSION_NAME": true,
-        "getCurrentTab": true,
-        "getLoginData": true,
-        "getTopLevelDomainFromUrl": true,
-        "GRAY_BUTTON_CLASS": true,
-        "GREEN_BUTTON": true,
-        "httpAuth": true,
-        "Icon": true,
-        "IGNORE_AUTOSUBMIT": true,
-        "IGNORE_FULL": true,
-        "IGNORE_NORMAL": true,
-        "IGNORE_NOTHING": true,
-        "importScripts": true,
-        "IMPROVED_DETECTION_PREDEFINED_SITELIST": true,
-        "initColorTheme": true,
-        "isEdge": true,
-        "isFirefox": true,
-        "keepass": true,
-        "keepassClient": true,
-        "kpActions": true,
-        "kpErrors": true,
-        "kpxc": true,
-        "kpxcAssert": true,
-        "kpxcBanner": true,
-        "kpxcCustomLoginFieldsBanner": true,
-        "kpxcEvent": true,
-        "kpxcFields": true,
-        "kpxcFill": true,
-        "kpxcForm": true,
-        "kpxcIcons": true,
-        "kpxcObserverHelper": true,
-        "kpxcPasswordGenerator": true,
-        "kpxcPasswordIcons": true,
-        "kpxcSites": true,
-        "kpxcTOTPAutocomplete": true,
-        "kpxcTOTPIcons": true,
-        "kpxcUI": true,
-        "kpxcUserAutocomplete": true,
-        "kpxcUsernameIcons": true,
-        "logDebug": true,
-        "logError": true,
-        "kpxcPasskeysUtils": true,
-        "ManualFill": true,
-        "matchesWithNodeName": true,
-        "MAX_AUTOCOMPLETE_NAME_LEN": true,
-        "MAX_OPACITY": true,
-        "MAX_TOTP_INPUT_LENGTH": true,
-        "menuContexts": true,
-        "MIN_INPUT_FIELD_OFFSET_WIDTH": true,
-        "MIN_INPUT_FIELD_WIDTH_PX": true,
-        "MIN_OPACITY": true,
-        "MIN_TOTP_INPUT_LENGTH": true,
-        "module": true,
-        "nacl": true,
-        "ORANGE_BUTTON": true,
-        "page": true,
-        "Pixels": true,
-        "PublicKeyCredential": true,
-        "PREDEFINED_SITELIST": true,
-        "RED_BUTTON": true,
-        "retrieveColorScheme": true,
-        "sendMessage": true,
-        "showNotification": true,
-        "siteMatch": true,
-        "slashNeededForUrl": true,
-        "SORT_BY_GROUP_AND_TITLE": true,
-        "SORT_BY_GROUP_AND_USERNAME": true,
-        "SORT_BY_MATCHING_CREDENTIALS_SETTING": true,
-        "SORT_BY_RELEVANT_ENTRY": true,
-        "SORT_BY_TITLE": true,
-        "SORT_BY_USERNAME": true,
-        "statusResponse": true,
-        "Tests": true,
-        "tr": true,
-        "trimURL": true
+        "_called": "writable",
```

Comment:
Is this intentional?

Reply:
It was, but now after double checking, that one is an Object that is never rewritten, even if its values are. Fixed.